facebook/Docusaurus
# Add Reaction Commerce to users

## Motivation

Add Reaction Commerce to users

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

☑️

## Test Plan

- Look at the Netlify deploy branch: https://deploy-preview-869--docusaurus-preview.netlify.com/en/users
- Click on the Reaction Commerce logo: https://docs.reactioncommerce.com/docs/intro.html
- Make sure it works
- 💯

## Screenshot

<img width="1440" alt="screen shot 2018-07-23 at 5 08 14 pm" src="https://user-images.githubusercontent.com/3673236/43109314-0b1cd49a-8e9b-11e8-892c-90268df5226d.png">


## Related PRs

none
